lib/ethereumex/config.ex

```diff
@@ -25,4 +25,25 @@ defmodule Ethereumex.Config do
   def request_timeout do
     Application.get_env(:ethereumex, :request_timeout, 5000)
   end
+
+  def setup_children do
+    setup_children(Application.get_env(:ethereumex, :client, :http))
```

Comment:
What do you think about setting up children directly in `Ethereumex` module and using `Config` module only for getting configuration params?